bitcoin/src/bip32.rs

```diff
@@ -37,12 +37,12 @@ const VERSION_BYTES_TESTNETS_PUBLIC: [u8; 4] = [0x04, 0x35, 0x87, 0xCF];
 const VERSION_BYTES_TESTNETS_PRIVATE: [u8; 4] = [0x04, 0x35, 0x83, 0x94];
 
 /// The old name for xpub, extended public key.
-#[deprecated(since = "0.31.0", note = "use xpub instead")]
-pub type ExtendendPubKey = Xpub;
```

Comment:
The type is already released so we can't really do this in case someone "fixed" the typo with `allow(deprecated)`. As I said, this needs to be a strict addition.

I would be fine with documenting that typo though.